airbyte-integrations/connectors/source-chargebee/unit_tests/resource/config/config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Why was this added?

Reply:
Good question. We didn't hit the required threshold for code coverage so I added some tests to verify that `run.py` could run a check and I needed a `config.json` fixture to do so